content/2021-09-20-brigade-2-reaches-third-beta-release.md

```diff
@@ -0,0 +1,77 @@
+
+## Other Integrations
+
+In addition to gateways, the team has two other integrations that we expect to be in at least beta states when v2 reaches GA. These are [Brigade Metrics](https://github.com/brigadecore/brigade-metrics) and the [Noisy Neighbor](https://github.com/brigadecore/brigade-noisy-neighbor), both of which we use to gain operational insight into our own v2 installation. Both of these are, again, largely thanks to Willie Yao, with guidance from project maintainers.
+
+## SDKs
+
+One reason we've been able to build so many integrations so quickly is that we took the time early on to build great SDKs to bind some of our favorite languages to the v2 APIs. We've actually somewhat surprised ourselves with how easy it is to create new integrations. Indeed, in some of the simpler cases, we've gone from prototype to an alpha release in just one day.
+
+If you'd like to build integrations of your own, have a look at:
+
+* [Brigade SDK for Go](https://github.com/brigadecore/brigade/tree/v2/sdk)
+* [Brigade SDK for JavaScript (and TypeScript)](https://github.com/brigadecore/brigade-sdk-for-js)
+
+Our [Brigade SDK for Rust]() remains a work-in-progress -- and an area where we'd gladly accept community assistance from fellow Rustaceans.
+
+## Community Building
+
+Brigade maintainers love open source and we love the open source community -- especially Brigade's own users. Unfortunately, we've observed community involvement in the project declining over the past year. Frankly, this was to be expected. History has shown us that when open source projects undertake major pivots, users are understandably reluctant to invest time and energy in either the pre-pivot versions or the unstable next gen version. That reluctance can also be compounded by stale, pre-pivot documentation.
+
+So, now's a good time to emphasize that although Brigade v2 remains in beta at this time, its APIs are stable and the team is committed to avoiding breaking changes at all cost. A complete documentation refresh is also well underway. In short, if you're one of our valued community members who has understandably taken a step back while the maintainers have executed this complex pivot, now is a great time to get involved again! And we'll be happy to set aside some great swag for anyone who wants to help kick the tires on a beta release or develop a new integration. More important than swag, the team is very interested in diversifying the ranks of maintainers, not only in terms of demographics, but also in terms of what companies are represented. The project's current state presents a prime opportunity to quickly become a maintainer of a CNCF project -- a great thing to have on your CV!
+
+If you'd like to collaborate with us, please do find us on GitHub or on our [Slack channel](https://kubernetes.slack.com/messages/C87MF1RFD).
```

Comment:
Perhaps `GitHub` can be a link as well? (To https://github.com/brigadecore/brigade)?

Reply:
Yup... will even link straight to the v2 branch.